VirtoCommerce.Storefront/Domain/Cart/CartConverter.cs

```diff
@@ -522,7 +522,7 @@ public static cartDto.ShoppingCart ToShoppingCartDto(this ShoppingCart cart)
                 result.LanguageCode = cart.Language.CultureName;
             }
             result.Addresses = cart.Addresses.Select(ToCartAddressDto).ToList();
-            result.Coupon = cart.Coupon != null ? cart.Coupon.Code : null;
+            result.Coupons = cart.Coupons != null ? cart.Coupons.Select(c => c.Code).ToList() : null;
```

Comment:
This line won't be compiled isn't?

Reply:
The "Coupons" field of "cartDto.ShoppingCart" object has a list of strings type:
   public IList<string> Coupons { get; set; }
Dto cart object is being sent to "api/carts" to save the cart.

Reply:
Can be simplified:
`result.Coupons = cart.Coupons?.Select(c => c.Code).ToList();`